pkg/users/git_test.go

```diff
@@ -53,6 +53,27 @@ func TestFindUserByLabel(t *testing.T) {
 	assert.Equal(t, labeleduserID, user.Spec.Login)
 }
 
+func TestFindUserByUpperCaseGitUserName(t *testing.T) {
+	t.Parallel()
+	resolver, _, err := prepare(t)
+	assert.NoError(t, err)
+	assert.NoError(t, err)
+	gituserID := "foo"
+
+	gitUser := gits.GitUser{
+		Login: gituserID,
```

Comment:
any reason you are not using "foo" directly? For symmetry reason and to make it easier to understand the test, I think it makes sense to put in "foo" directly here.